certified-connectors/TOPdesk/apiDefinition.swagger.json

```diff
@@ -0,0 +1,11510 @@
+            "description": "How many incidents should be returned max. Default is 10.\n",
+            "minimum": 1,
+            "maximum": 10000,
+            "in": "query",
+            "type": "integer"
+          },
+          {
+            "name": "sort",
+            "in": "query",
+            "type": "string",
+            "description": "The sort order of the returned incidents. Incidents can be ordered by most of the fields. But for best performance one should order by one of the following fields: callDate, creationDate,\nmodificationDate, targetDate, closedDate or id. It's faster to order by 1 field only. To specify if the order should be ascending or descending,\nappend \":asc\" or \":desc\" to the field name. Multiple columns can be specified by comma-joining the\norderings. Example: sort=tragetDate:asc,creationDate:desc.\nFields not allowed for sorting are: externalLinks, escalationStatus, action, attachments, partialIncidents, partialIncidents.link\n"
+          },
+          {
+            "name": "query",
+            "description": "A FIQL string to select which incidents should be returned. (See [Tutorial](https://developers.topdesk.com/tutorial.html#query))\n",
+            "in": "query",
+            "type": "string"
+          },
+          {
+            "name": "fields",
+            "description": "A comma-separated list of which fields should be returned. By default all fields will be returned. (slow)\n",
+            "in": "query",
+            "type": "string"
+          },
+          {
+            "name": "dateFormat",
+            "description": "Format of date fields in json. When set to iso8601 dates will be sent in the form of '2020-10-01T14:10:00Z'.\nOtherwise old date format will be used: '2020-10-01T14:10:00.000+0000'\n",
+            "in": "query",
+            "type": "string",
+            "enum": [
+              "iso8601"
+            ]
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/Incident"
+              }
+            }
+          },
+          "204": {
+            "description": "No (matching) incidents found"
+          },
+          "206": {
+            "description": "Success, but there are more incidents",
+            "headers": {
+              "x-next-page-url": {
+                "type": "string",
+                "format": "url",
+                "description": "A link that returns the next list of incidents"
+              }
+            },
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/Incident"
+              }
+            }
+          },
+          "400": {
+            "description": "_Bad Request_\n<ul>\n  <li>The parameters of the request were malformed</li>\n</ul>\n"
```

Comment:
@CJusjong The description values of response codes can be a simple text as there are no multiple list items to use html tags.